specification/search/resource-manager/Microsoft.Search/preview/2025-02-01-preview/search.json

```diff
@@ -2568,6 +2616,23 @@
           "readOnly": true,
           "type": "string",
           "description": "A system generated property representing the service's etag that can be for optimistic concurrency control during updates."
+        },
+        "upgradeAvailable": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Indicates whether or not the search service has an upgrade available."
+        },
+        "serviceCreatedDate": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The date and time the search service was created."
+        },
+        "serviceUpgradeDate": {
```

Comment:
serviceUpgradeDate or serviceUpgradedDate?. In the samples you use serviceUpgradedDate.